checker/jdk/Makefile.jdk

```diff
@@ -30,8 +30,8 @@ build: classes
 
 ############## Validate annotated jdk against missing methods ##############
 
-SOURCES = $(shell find src/java src/javax -name '*.java')
-# CLASSES = $(shell find src/java src/javax -name '*.class')
+SOURCES = $(shell find src/java src/javax src/org/w3c src/org/xml -name '*.java')
```

Comment:
Good point about `com/google`! They will soon be moved to a dedicated repository, if I'm not mistaken here:
https://github.com/typetools/guava
So we don't need to worry about them.